extensions/linz/README.md

```diff
@@ -107,12 +107,12 @@ This expands on the [provider object in the STAC spec](https://github.com/radian
 
 These fields apply to assets within both items and collections.
 
-| Field Name              | Type   | Description                                                                                                                     |
-| ----------------------- | ------ | ------------------------------------------------------------------------------------------------------------------------------- |
-| assets/\*/created       | string | **REQUIRED**. Creation date and time of the asset, in UTC.                                                                      |
-| assets/\*/updated       | string | **REQUIRED**. Date and time the asset was last updated, in UTC.                                                                 |
-| assets/\*/file:checksum | string | **REQUIRED**. See [reference](https://github.com/stac-extensions/file/blob/v2.0.0/README.md#checksums).                         |
-| linz:language           | string | **REQUIRED**. [RFC 5646 language tag](https://datatracker.ietf.org/doc/html/rfc5646), for example `en-NZ` (New Zealand English) |
+| Field Name              | Type   | Description                                                                                                       |
+| ----------------------- | ------ | ----------------------------------------------------------------------------------------------------------------- |
+| assets/\*/created       | string | **REQUIRED**. Creation date and time of the asset, in UTC.                                                        |
+| assets/\*/updated       | string | **REQUIRED**. Date and time the asset was last updated, in UTC.                                                   |
+| assets/\*/file:checksum | string | **REQUIRED**. See [reference](https://github.com/stac-extensions/file/blob/v2.0.0/README.md#checksums).           |
+| linz:language           | string | [RFC 5646 language tag](https://datatracker.ietf.org/doc/html/rfc5646), for example `en-NZ` (New Zealand English) |
```

Comment:
Can you add 'For data that includes text, recommended when shared externally.' or something like that. I can't remember if we had similar wording elsewhere for 'recommdended' data.

Reply:
How about "Required for assets which include text when the collection is shared externally."?